src/backend/distributed/commands/schema_based_sharding.c

```diff
@@ -258,3 +258,19 @@ ErrorIfTenantTable(Oid relationId, char *operationName)
 							   operationName)));
 	}
 }
+
+
+/*
+ * ErrorIfTenantSchema errors out with the given operation name,
+ * if the given schema is a tenant schema.
+ */
+void
+ErrorIfTenantSchema(Oid nspOid, char *operationName)
+{
+	if (IsTenantSchema(nspOid))
+	{
+		ereport(ERROR, (errmsg("%s is not allowed for %s because it is a tenant schema",
```

Comment:
We update messages to not contain `tenant schema` phrase after we change the udf namingsin PR https://github.com/citusdata/citus/pull/6933. (e.g. citus_schema_distribute instead of citus_tenant_schema_set)

Can you change the message like
```c
"%s is not allowed for %s because it is a distributed schema"
```